src/pages/home/sidebar/SidebarLinks.tsx

```diff
@@ -91,10 +91,10 @@ function SidebarLinks({insets, optionListItems, isLoading, priorityMode = CONST.
                     onSelectRow={showReportPage}
                     shouldDisableFocusOptions={shouldUseNarrowLayout}
                     optionMode={viewMode}
-                    onFirstItemRendered={App.setSidebarLoaded}
+                    onFirstItemRendered={setSidebarLoaded}
                 />
                 {!!isLoading && optionListItems?.length === 0 && (
-                    <View style={[StyleSheet.absoluteFillObject, styles.appBG]}>
+                    <View style={[StyleSheet.absoluteFillObject, styles.appBG, styles.mt3]}>
```

Comment:
What does the `.mt3` do here exactly? Can you share a before and after screenshot?